apps/prairielearn/src/pages/instructorCopyTemplateCourseQuestion/instructorCopyTemplateCourseQuestion.ts

```diff
@@ -39,8 +39,8 @@ router.post(
       }),
     );
 
-    if (result.course.template_course === false) {
-      throw new error.HttpStatusError(400, 'The given course is not a template course.');
+    if (!(result.course.template_course || result.question.shared_publicly_source)) {
+      throw new error.HttpStatusError(400, 'Copying this question is not permitted');
     }
```

Comment:
Is there a reason to have a separate endpoint for copying from a template course and copying a publicly shared question? Or should I just rename this endpoint to `instructorCopyPublicQuestion`, and use it both for publicly shared questions and template course questions?

Reply:
and what should I do with the URL? its currently `'/pl/course/:course_id(\\d+)/copy_template_course_question'`

Reply:
changed to `copy_public_question`